src/engine/api/json_factory.cpp

```diff
@@ -193,8 +193,8 @@ util::json::Object makeStepManeuver(const guidance::StepManeuver &maneuver)
             detail::instructionModifierToString(maneuver.instruction.direction_modifier);
 
     step_maneuver.values["location"] = detail::coordinateToLonLat(maneuver.location);
-    step_maneuver.values["bearing_before"] = std::round(maneuver.bearing_before);
-    step_maneuver.values["bearing_after"] = std::round(maneuver.bearing_after);
+    step_maneuver.values["bearing_before"] = std::fmod(std::round(maneuver.bearing_before), 360);
+    step_maneuver.values["bearing_after"] = std::fmod(std::round(maneuver.bearing_after), 360);
```

Comment:
Yeah, good idea.  Because the logic was used in two functions, I've created an inline `detail::roundAndClampBearing()` that I've re-used.

The `bearings` array previously was not explicitly rounded, but I think the values used there pre-rounded deeper in the stack, so we should see no change of behaviour.